integration/testdata/envoy_bootstrap.yaml

```diff
@@ -0,0 +1,39 @@
+# Base config for a split xDS management server on 18000, admin port on 19000
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address:
+      address: 127.0.0.1
+      port_value: 19000
+dynamic_resources:
+  cds_config:
+    api_config_source:
+      api_type: GRPC
+      grpc_services:
+      - envoy_grpc:
+          cluster_name: xds_cluster
+      set_node_on_first_message_only: true
```

Comment:
https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/config_source.proto In our design we always expect xds relay to have this `set_node_on_first_message_only` to be false?

Reply:
@jessicayuen wdyt about the constraint in our system?

Reply:
its not a concern since gcp handles it https://github.com/envoyproxy/go-control-plane/blob/master/pkg/server/v2/server.go#L301-L305